update to project.toml

SimonDanisch opened the pull request:
@Gnimuc do you think the version is a good idea?
Maybe we can drop the minor version from the vulkan spec, assuming that it's irrelevant to something as broad as the julia wrapper, and use it for versioning the wrapper?

SimonDanisch committed:
update to project.toml

Gnimuc commented:
> Maybe we can drop the minor version from the vulkan spec

Did you mean the third version number a.k.a the patch version? 

As per discussed on Slack, Kristoffer proposed an alternative way:
```julia
v1.1.1140
v1.1.1141
...
```
This assumes the third version number never goes that high in practice.

Gnimuc commented on Project.toml:
```diff
@@ -0,0 +1,15 @@
+
+[compat]
+julia = "1"
+
```
```suggestion
CEnum = "~0.2.0"
```

Gnimuc commented:
@SimonDanisch, I just found Stefan also suggest to decouple the third version number:

> if the library being wrapped is following semver even slightly, then only the first two version numbers should matter for the API, in which case it might be ok for the third number to be decoupled from the patch number of the thing being wrapped.

Gnimuc commented:
However, [The Vulkan-Headers README](https://github.com/KhronosGroup/Vulkan-Headers#version-tagging-scheme ) suggest to use `sdk-<version>.<patch>` format (e.g., sdk-1.1.92.0).

Gnimuc commented:
The ultimate solution would be to make Julia's package registry support ~~the 4-th version number~~pre-release identifier, I filed a feature request here https://github.com/JuliaLang/Pkg.jl/issues/1288.

SimonDanisch committed:
Update Project.toml

Co-Authored-By: Gnimuc <qiyupei@gmail.com>

SimonDanisch committed:
update to reflect version of julia wrapper 

version scheme:
```
SDKVersionX.X.WrapperVersionX
```
So this is the Vulkan SDK v1.1 with Wrapper version v1.1

SimonDanisch commented:
Ok, I'm not gonna wait for that to happen :D 
I just changed version to `version = "1.1.11"`, as SDK 1.1 with wrapper 1.1 (old wrapper version was 1.0, so I bumped that to 1.1) ;)

Gnimuc commented:
OK, let's merge this. 

BTW, what's your opinion on #25? If we won't limit the scope of this package to Vulkan Core API, this is the best chance to change the package name before registering the package.

SimonDanisch commented:
Ok, I made a comment! I'll merge, and if you're fine with the comment, lets tag!
